action.yml

```
@@ -8,6 +8,10 @@ inputs:
     description: Label name for epic issues
     required: true
     default: epic
+  auto-close-epic:
+    description: Auto closes epic issue when all referenced issues are closed
+    required: false
+    default: true
```

I'd rather keep the default to false so that we don't change the current behavior.

```suggestion
    default: false
```